### PR TITLE
[4.13] CORS-2420 Compact 3-node Clusters on Public Cloud (GCP)

### PR DESCRIPTION
**Pull request by jianli-wei**
1. The 4.13 epic: [CORS-2420 Compact 3-node Clusters on Public Cloud (GCP)](https://issues.redhat.com/browse/CORS-2420)
2. Add workflow `cucushift-installer-rehearse-gcp-ipi-compact-3-nodes`

### CODE REVIEW

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-gcp

**Comment by jianli-wei** [comment]
@jianlinliu @gpei please help review, thanks! 

FYI
```
$ gcloud compute instances list --filter='name~ci-op-n1wbk6xc'
NAME                                 ZONE           MACHINE_TYPE   PREEMPTIBLE  INTERNAL_IP  EXTERNAL_IP  STATUS
ci-op-n1wbk6xc-0f951-nq2z4-master-0  us-central1-a  e2-standard-8               10.0.0.3                  RUNNING
ci-op-n1wbk6xc-0f951-nq2z4-master-1  us-central1-b  e2-standard-8               10.0.0.5                  RUNNING
ci-op-n1wbk6xc-0f951-nq2z4-master-2  us-central1-c  e2-standard-8               10.0.0.4                  RUNNING
$

File: ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.metadata.json:2]: How about rename `compact-3-nodes` to `compact` to keep consistent with azure.

File: ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-compact-3-nodes
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-provision
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.yaml:5]: @jinyunma when you have time, can you help update azure-ipi-compact workflow similarly, seem like we can this is simpler than introducing provision chain.

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-gcp

File: ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.yaml (hunk 2 above)
Comment (jinyunma) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.yaml]: ah, okay, will update on azure platform.

**Pull request merged**